### PR TITLE
Natasha/inf scroll

### PR DESCRIPTION
**Pull request by natashaarmbrust**
- adds inifnite scroll functionality to feed
- refactors loading animators 
- reverses sorted order for feed
- Known issues: search still needs to be refactored

### CODE REVIEW

File: Podcast/FeedViewController.swift [hunk 1]
```diff
@@ -80,20 +81,16 @@ class FeedViewController: ViewController {
...
         
         fetchFeedEndpointRequest.success = { (endpoint) in
             guard let feedElementsFromEndpoint = endpoint.processedResponseValue as? [FeedElement] else { return }
             for feedElement in feedElementsFromEndpoint {
                 self.feedSet.insert(feedElement)
             }
-            
-            self.feedElements = self.feedSet.sorted { (fe1,fe2) in fe1.time < fe2.time }
+
+            self.feedElements = self.feedSet.sorted { (fe1,fe2) in fe1.time > fe2.time }
```
Comment (doctor3w) [inline Podcast/FeedViewController.swift:92]: lmao nice

**Review by doctor3w — APPROVED** [review]
LGTM

File: Podcast/ActivitityIndicatorView.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import UIKit
+import NVActivityIndicatorView
+
+class LoadingAnimatorUtilities {
+
+    static var loadingAnimatorSize: CGFloat = 30
+
+    static func createLoadingAnimator() -> NVActivityIndicatorView {
+        let color: UIColor = .sea
+        let type: NVActivityIndicatorType = .lineScalePulseOut
```
Comment (mindylou) [inline Podcast/ActivitityIndicatorView.swift:18]: ooooh this one looks nice

**Review by mindylou — APPROVED** [review]
lgtm!

**Pull request merged**